README.md

```diff
@@ -21,6 +21,7 @@ The required environment variables for this API are the following:
 
 ````bash
 export FLASK_ENV=development
+export YELP_API_KEY=<insert>
```

Comment:
can we drop the `export` keyword, `autoenv` doesn't require it and we can copy this straight into a server's configuration if the `export` keyword is removed.